src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/SeedCharacteristicsTest.kt

```diff
@@ -0,0 +1,154 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.SeedFileType
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+class SeedCharacteristicsTest : SeedTestBase() {
+  @Test
+  fun `Attempting to seed characteristic with missing characteristic_name field fails and logs error`() {
+    characteristicRepository.deleteAll() // can't seem to do this in a BeforeEach setUp function
+
+    withCsv(
+      "invalid-characteristics-missing-name",
+      "characteristic_property_name,characteristic_name,service_scope,model_scope\n" +
+        "hasWideDoor,Is the door to this room at least 900mm wide?,approved-premises,room\n" +
+        "hasWideDoor,,temporary-accommodation,room\n"
+    )
+
+    seedService.seedData(SeedFileType.characteristics, "invalid-characteristics-missing-name")
+
+    Assertions.assertThat(characteristicRepository.count()).isEqualTo(0)
```

Comment:
Would import `assertThat` rather than `Assertions` so these read a bit more naturally:

- Add `import org.assertj.core.api.Assertions.assertThat` to the imports at the top of the file
- Remove all of the `Assertions.`

Reply:
thanks, I've done that.